cmd/nerdctl/container_run_test.go

```diff
@@ -467,3 +468,21 @@ func TestRunWithTtyAndDetached(t *testing.T) {
 	withTtyContainer := base.InspectContainer(withTtyContainerName)
 	assert.Equal(base.T, 0, withTtyContainer.State.ExitCode)
 }
+
+func TestRunWithDetachKeys(t *testing.T) {
+	t.Parallel()
+
+	base := testutil.NewBase(t)
+	if testutil.GetTarget() == testutil.Nerdctl {
+		testutil.RequireDaemonVersion(base, ">= 1.6.0-0")
```

Comment:
why this check ?

Reply:
Removed. I copied from [`TestRunStdin`](https://github.com/containerd/nerdctl/blob/225a70bdc3b93cdb00efac7db1ceb50c098a8a16/cmd/nerdctl/container_run_test.go#L223-L225), but it turned out this check is only needed when `-t` is not specified with `-i` ([source](https://github.com/containerd/nerdctl/blob/225a70bdc3b93cdb00efac7db1ceb50c098a8a16/pkg/taskutil/taskutil.go#L97-L99)). Thanks for pointing it out!